Examples/Examples-iOS/IGListKitExamples-UITests/StoryboardViewControllerUITests.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  StoryboardViewControllerUITests.swift
+//  IGListKitExamples
+//
+//  Created by Huylens on 2017/7/1.
+//  Copyright © 2017年 Instagram. All rights reserved.
+//
```

Comment:
We should move stuff to the layout tests, but for future's sake we require the FB copyright header in all source, test, and example files.